### DISCUSSION TITLE
How to send batch emails for proposals?

### DISCUSSION THREAD

**Discussion by EmmanuelDemey** [conference-hall/conference-hall#299] [Q&A] [answered]
Hi,

Thank you very much for this new version!

I have a quick question. We can reject a list of talks without any issue, but are we able to send rejection emails with a single click?

The only option we found for sending emails was from the individual proposal page. We'd prefer not to have to manually click a button on every rejected proposal page. 😄

**Comment by bpetetot** [comment, marked as answer]
Hi @EmmanuelDemey ,

Yes, you can do it in the **"Publication" tab** (Only available for conference).

You can publish results and send emails for all accepted proposals or all rejected proposals.

![image](https://github.com/user-attachments/assets/5576f97d-b460-4d39-8f45-78eacca86ba8)

**Comment by EmmanuelDemey** [comment]
Thanks @bpetetot . The "Publish all" button is not easy to understand for me :D

**Reply by bpetetot** [reply]
Ah... 😅 Do you have suggestions?

"Send results", "Send results to speakers"...

I don't want to put the "email" word because, it does not only send emails, it publishes the CFP results to speakers in Conference Hall.

**Reply by EmmanuelDemey** [reply]
Publish et Notify all?
